react/useAddressRules.tsx

```diff
@@ -23,7 +23,7 @@ const useAddressRules = () => {
           },
         }),
         {}
-      ) ?? {},
+      ),
```

Comment:
Although I agree that this change makes sense, it would be a breaking change, right? Have you checked if this is going to break somewhere?

Reply:
I did, most of the apps that use this function just pass the result from it to [`address-context` Provider](https://github.com/vtex-apps/address-context/blob/master/react/AddressContext.tsx#L30), which has a default value for the rules to be an empty object (which I checked that if I pass undefined to the rules prop, it will correctly use the empty object instead), so it shouldn't break anyone 😅

But I linked it in [this workspace](https://lucas--checkoutio.myvtex.com/) just to be sure.